Fix unistd.h include

Fixes build error

source4/heimdal/lib/asn1/asn1_err.c:47:23: error: 'link' redeclared as different kind of symbol
 static struct et_list link = { 0, 0 };

Downloaded from
https://github.com/openwrt/packages/blob/master/net/samba4/patches/006-samba-4-10-musl_rm_unistd_incl.patch

Upstream bug report: https://bugzilla.samba.org/show_bug.cgi?id=13856

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

--- a/lib/replace/replace.h
+++ b/lib/replace/replace.h
@@ -162,10 +162,6 @@
 #include <bsd/unistd.h>
 #endif
 
-#ifdef HAVE_UNISTD_H
-#include <unistd.h>
-#endif
-
 #ifdef HAVE_STRING_H
 #include <string.h>
 #endif
